doc/source/reference/random/new-or-different.rst

```diff
@@ -36,7 +36,7 @@ Feature            Older Equivalent     Notes
                                         Many other distributions are also
                                         supported.
 ------------------ -------------------- -------------
-``Generator().``   ``randint``,         Use the ``endpoint`` kwarg to adjust
+``Generator.``     ``randint``,         Use the ``endpoint`` kwarg to adjust
```

Comment:
Both here and above,  `Generator` -> `default_gen()`

Reply:
I disagree. People really shouldn't be calling `default_gen().<method>()`. They should make an instance separately, then call methods on it. I'll drop the `()` to emphasize that these are references to the methods, not actual call examples.

Reply:
ok